app/uwsgi.ini

```diff
@@ -2,3 +2,7 @@
 module = server.wsgi
 callable = app
 enable-threads = true
+reload-mercy = 3600
+worker-reload-mercy = 3600
+http-timeout = 3600
```

Comment:
`set the maximum time (in seconds) we wait for workers and other processes to die during reload/shutdown`

3600s / 60s = 60 minutes. That seems to be a really long time if we need to restart the server for some reason?